smart_led/config/config.exs

```diff
@@ -26,3 +26,18 @@ config :logger, backends: [RingLogger]
 if Mix.target() != :host do
   import_config "target.exs"
 end
+
+
+# https://elixirschool.com/en/lessons/advanced/nerves/#setting-up-networking
+# Statically assign an address
+config :nerves_network, :default,
```

Comment:
Should this be in `config/target.exs` so it only gets configured on the target device?

Also, I think nerves doesn't use `nerves_network` anymore and has switched to `vintage_net`, which means the config will be slightly different.

Reply:
Yeah, it seems the there's a default `vintage_net` config in `config/target.exs`.

I don't know why they changed it, `nerves_network` seems a lot simpler.

For reference, this is how I setup VintageNet on a previous project: https://github.com/th0mas/Fancy-Lights/blob/master/fancy_lights_firmware/config/target.exs#L44

Reply:
@th0mas thanks for sharing direct link to your working code. 🚀